server/sandbox_run.go

```diff
@@ -306,23 +322,21 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		}
 	}
 
-	err = g.SaveToFile(filepath.Join(podSandboxDir, "config.json"), generate.ExportOptions{})
+	saveOptions := generate.ExportOptions{}
+	mountPoint, err := s.storage.StartContainer(id)
```

Comment:
The directory where we're storing the config.json file should get removed along with rest of the pod sandbox storage whenever it gets deleted, so I think this is fine.